README.md

```diff
@@ -15,6 +15,12 @@ In particular, the output code requires the following [Wasm extensions](https://
 - [the tail-call extension](https://github.com/WebAssembly/tail-call/blob/main/proposals/tail-call/Overview.md)
 - [the exception handling extension](https://github.com/WebAssembly/exception-handling/blob/master/proposals/exception-handling/Exceptions.md)
 
+OCaml 5.x code using effect handlers can be compiled in two different ways:
+One can enable the CPS transformation from `js_of_ocaml` by passing the
+`--enable=effects` flag. Without the flag `wasm_of_ocaml` will instead emit code
+utilizing
+- [the JavaScript-Promise Integration extension](https://github.com/WebAssembly/js-promise-integration/blob/main/proposals/js-promise-integration/Overview.md)
```

Comment:
It feels like we can name things better here. I understand reading this comment that effects are enabled by default and the choice here is whether to use `jspi` or `cps`. So `--enable=effects` option feels like a misnomer. Would it be better to change the behaviour such that there are two options `--enable=effects_cps` and `--enable=effects_jspi` with `--enable=effects_jspi` being the default behaviour?

Reply:
For now, I have just reused the flag that enables the CPS transformation in Js_of_ocaml.
But indeed, we should use a better name at some point. Probably even have a specific flag: `--effects=cps/jspi/double-translation`.